### PR TITLE
[dev] AB#41363 - Button Dropdown Active State

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

File: src/colors/cyan.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const cyan = {
     500: '#00aeef',
-    600: '#0093c6',
+    600: '#1c93c4',
```
Comment (groberts314) [inline src/colors/cyan.js:3]: This is a sweeping change, right?  Just confirming it is what is intended?

**Review by groberts314 — APPROVED** [review]
LGTM

File: src/colors/cyan.js (hunk 1 above)
Comment (cbrewer1689) [reply src/colors/cyan.js:3]: Yes, this was intended to match what design has in the docs.

Comment (groberts314) [reply src/colors/cyan.js:3]: Plus, not much of a difference

cyan_600_switcheroo

**Pull request merged**